### PR TITLE
OHRM5X-2140: Develop employee picture - add etag caching

### PR DESCRIPTION
**Pull request by devishke-orange**
PR contains:
- OHRM5X-2140: [BE] Cache employee profile picture

### CODE REVIEW

File: src/plugins/orangehrmPimPlugin/Controller/File/EmployeePictureController.php [hunk 1]
```diff
@@ -61,7 +58,7 @@ public function handle(Request $request)
             $empPicture = $this->getEmployeePictureService()->getEmpPictureByEmpNumber($empNumber);
```
Comment (RajithaKumara) [inline src/plugins/orangehrmPimPlugin/Controller/File/EmployeePictureController.php:58]: I want to minimize this call, as it goes to DB. want to reduce this call and cache the profile picture.

**Review by RajithaKumara — APPROVED** [review]

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -62,7 +62,7 @@ jobs:
             -e LDAP_CONFIG_ADMIN_PASSWORD=admin \
             -e LDAP_SKIP_DEFAULT_TREE=yes \
             -v $PWD/ldifs:/ldifs \
-            bitnami/openldap:latest
```
Comment (devishke-orange) [inline .github/workflows/test.yml:65]: temp change due to test failure on latest LDAP container

File: .github/workflows/test.yml [hunk 3]
```diff
@@ -62,7 +62,7 @@ jobs:
...
+            bitnami/openldap:2.6.4
```
Comment (devishke-orange) [inline .github/workflows/test.yml:65]: temp change due to test failure on latest LDAP container

File: src/plugins/orangehrmPimPlugin/Controller/File/EmployeePictureController.php (hunk 1 above)
Comment (devishke-orange) [reply src/plugins/orangehrmPimPlugin/Controller/File/EmployeePictureController.php]: https://github.com/orangehrm/orangehrm/pull/1674/files#diff-7a6b406d8bb4fdbbe35be9fa188826a121e49244e2da957cdeb91dc322f0328fR58-R67

DB call will not happen if the eTag is cached and the response is not modified

**Pull request merged**